src/Provider/ImageProviderInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Provider;
+
+interface ImageProviderInterface extends FileProviderInterface
```

Comment:
Shouldnt the FileProvider have the FileProviderInterface instead of here extending it? What is the better option?

Reply:
In ImageProvider using of FileProviderInterface is possible. People who write own ImageProvider should be force to implement it becouse Image is File too.